src/Features/Core/Portable/CodeFixes/ICodeFixService.cs

```diff
@@ -12,7 +13,8 @@ internal interface ICodeFixService
     {
         Task<ImmutableArray<CodeFixCollection>> GetFixesAsync(Document document, TextSpan textSpan, bool includeSuppressionFixes, CancellationToken cancellationToken);
         Task<CodeFixCollection> GetDocumentFixAllForIdInSpan(Document document, TextSpan textSpan, string diagnosticId, CancellationToken cancellationToken);
+        Task<Document> ApplyCodeFixesForSpecificDiagnosticId(Document document, string diagnosticId, IProgressTracker progressTracker, CancellationToken cancellationToken);
```

Comment:
does ICodeFixService have any other API that apply something? jsut asking.

Reply:
I don't see any other API in ICodeFixService  that apply changes...